Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -22,6 +22,10 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         }
     }
 
+    class MyViewController: UIViewController {
+        var foo: String?
```

Comment:
Replace this property with an `IBOutlet` that you set in the storyboard, and then call `XCTAssertNotNil` in the test. I think that will more correctly ensure that the new function works correctly.